lib/vanity/adapters/mock_adapter.rb

```diff
@@ -137,9 +137,11 @@ def ab_add_participant(experiment, alternative, identity)
         alt[:participants] << identity
       end
 
-      def ab_seen(experiment, identity, alternative)
-        if ab_assigned(experiment, identity) == alternative.id
-          alternative
+      def ab_seen(*args)
```

Comment:
Hm, what do you think about keeping the arguments expanded, e.g. `experiment, identity, alternative_id` or `experiment, identity, alternative_or_id`? Condensing down to the splat might make the documentation harder to follow.